### DISCUSSION TITLE
Question about running the mace/cli/run_train.py

### DISCUSSION THREAD

**Discussion by zjucm** [ACEsuit/mace#684] [Q&A]
dear author and friends：
after downloading the MACE with "pip install mace-torch" ，i would like to running the given example on fine-tuning with train.xzy and test.xzy provided , but it keep sending errors "raceback(most recent calllast)File "/root/macetest/test.py",line 22,in <module>from mace.tools.model script utils import configure modelModuleNotFoundError: No module named 'mace.tools.model script utils'" ,how can i deal with it? I would more than appreciated if this problem solved! here is the full message after i bash my job.sh.

/root/miniconda3/envs/mace/lib/python3.12/site-packages/e3nn/o3/_wigner.py:10: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  _Jd, _W3j_flat, _W3j_indices = torch.load(os.path.join(os.path.dirname(__file__), 'constants.pt'))
Traceback (most recent call last):
  File "/root/macetest/test.py", line 22, in <module>
    from mace.tools.model_script_utils import configure_model
ModuleNotFoundError: No module named 'mace.tools.model_script_utils'

**Comment by ilyes319** [comment]
Can you check that you are running the script in the correct python environement? From the error, it would seem that you are not loading the env correct before running it. If you share your job.sh, I could help more.

**Reply by zjucm** [reply]
Of course,first i download python 3.12.4 and numpy 1.26.4 in my new environement, then i use the pip install mace-torch,and here is my job.sh :

#!/bin/bash
#SBATCH --job-name=cm
#SBATCH --nodes=1
#SBATCH --gres=gpu:2
#SBATCH -e err
#SBATCH --ntasks=16
source ~/.bashrc
conda activate mace
python test.py
--name="MACE"
--foundation_model="small"
--train_file="train.xyz"
--valid_fraction=0.05
--test_file="test.xyz"
--energy_weight=1.0
--forces_weight=1.0
--E0s="average"
--lr=0.001
--scaling="rms_forces_scaling"
--batch_size=4
--max_num_epochs=10
--ema
--ema_decay=0.99
--amsgrad
--default_dtype="float32"
--device=cuda
--seed=3

the test.py is mace/cli/run_train.py

**Reply by zjucm** [reply]
ok,thanks,my problem is solved!
